src/module/combat/combat.js

```diff
@@ -552,6 +552,153 @@ export class CombatSFRPG extends Combat {
         return this.getIndexOfFirstUndefeatedCombatant() === null;
     }
 
+    average(array) {
+        let total = 0;
+        let count = 0;
+        jQuery.each(array, function(index, value) {
+            total += value;
+            count++;
+        });
+        return total / count;
```

Comment:
I assure you you don't need jQuery for this 😄
```js
const average = (array) => array.reduce((total, value) => total + value), 0) / array.length;
```
We also don't need to add this function to the class itself, it can be declared inside the `calculateAPL` function to keep it local

Reply:
Cool, yeah this seems a lot simpler. Implemented.